### PR TITLE
generate batch script entryfile for windows

### PR DESCRIPTION
**Pull request by johnnychen94**
This doesn't fix every issue I've met when using Comonicon on Windows, but I believe it's a good first step.

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name = "Comonicon"
 uuid = "863f3e99-da2a-4334-8734-de3dacbe5542"
 authors = ["Roger-luo and contributors"]
-version = "1.0.5"
+version = "1.0.6"
```
Comment (Roger-luo) [inline Project.toml:4]: no need to bump version in the PR, I'll do that when I release a tag

**Review by Roger-luo — COMMENTED** [review]
Thanks! can you also add a windows CI so the tests run in this PR (with allow fail option true)?

**Comment by Roger-luo** [comment]
I'm good to merge this PR since it's not breaking any CIs, I'm wondering what you think would be the next step? e.g how should people add this to their path on windows?

**Comment by johnnychen94** [comment]
bump @Roger-luo

**Pull request merged**